migrations/versions/e1b6d52fd6ea_.py

```diff
@@ -0,0 +1,22 @@
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'e1b6d52fd6ea'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'profile',
```

Comment:
convention to the singular?

Reply:
Do we have a convention? Singular works until you need a `user` table.

Reply:
singular good for me as `user` is not a domain term